service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderService.java

```diff
@@ -114,10 +114,15 @@ public UploadDraftOrdersData getDraftsInfo(CaseData caseData) {
                     .map(Element::getValue)
                     .orElseThrow(() -> new CMONotFoundException("CMO for related hearing could not be found"));
 
-                newEventDataBuilder.previousCMO(cmo.getOrder())
-                    .cmoSupportingDocs(cmo.getSupportingDocs());
+                newEventDataBuilder.previousCMO(cmo.getOrder());
```

Comment:
vv minor but there is an unused field `FeatureToggleService` at the top

Reply:
removed it